tools/idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/ui/screen/settings/SettingsScreen.kt

```diff
@@ -19,16 +19,16 @@ import io.github.composegears.valkyrie.ui.foundation.ScrollableTabRow
 import io.github.composegears.valkyrie.ui.foundation.TopAppBar
 import io.github.composegears.valkyrie.ui.screen.settings.tabs.AboutSettingsScreen
 import io.github.composegears.valkyrie.ui.screen.settings.tabs.GeneralSettingsScreen
-import io.github.composegears.valkyrie.ui.screen.settings.tabs.export.ImageVectorExportSettingsScreen
-import io.github.composegears.valkyrie.ui.screen.settings.tabs.preview.ImageVectorPreviewSettingsScreen
+import io.github.composegears.valkyrie.ui.screen.settings.tabs.importSettings.ImageVectorImportSettingsScreen
+import io.github.composegears.valkyrie.ui.screen.settings.tabs.previewSettings.ImageVectorPreviewSettingsScreen
 
 val SettingsScreen by navDestination<Unit> {
     val navController = navController()
 
     val tabs = remember {
         listOf(
             TabItem(name = "General", screen = GeneralSettingsScreen),
-            TabItem(name = "Export", screen = ImageVectorExportSettingsScreen),
+            TabItem(name = "Import", screen = ImageVectorImportSettingsScreen),
```

Comment:
I suggest to rename this to `Generator` due to these settings related to code gen settings.
Also related code